components/helptext/src/styles/tokens.scss

```diff
@@ -0,0 +1,9 @@
+@import "@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables";
+
+:host {
+  --ds-auro-help-text-color: var(--ds-color-text-tertiary-default, #{$ds-color-text-tertiary-default});
+}
+
+:host([error]) {
```

Comment:
This should be moved into a `color.scss` file to keep consistent with all other components